Choreographer_30FPS/app/src/main/jni/Common/ndk_helper/gl3stub.h

```diff
@@ -0,0 +1,659 @@
+#ifndef __gl3_h_
+#define __gl3_h_
```

Comment:
wondering if we could make it vulkan style:  include gl3.h [ndk's copy] and only keep extras in this file, may you help to see if it is possible?